pyteck/eval_model.py

```diff
@@ -17,11 +17,11 @@
     print('Warning: YAML must be installed to read input file.')
     raise
 
-from pyked.chemked import ChemKED, DataPoint
+from pyked.chemked import ChemKED
 
 # Local imports
 from .utils import units
-from .simulation import Simulation
+from .simulation import AutoIgnitionSimulation as Simulation
```

Comment:
In general I'm not sure I like the idea of importing something under a different name - could likely confuse people down the line. Is there a good reason for it?